conf/iridanext.config

```diff
@@ -37,7 +38,7 @@ iridanext {
             csv {
```

Comment:
In this file, directly underneath the `rename` section, can you add a `keep` parameter that lists the column names in the CSV file to keep for metadata?

That is, something like:

```
keep = [
    "Quality Module",
    "Predicted Phenotype",
    ...
]
```

As an example, see the mikrokondo configuration: https://github.com/phac-nml/mikrokondo/blob/main/conf/irida_next.config#L137

The reason is that I noticed the "Isolate ID" is now being saved in the metadata in IRIDA Next, and I'd like to exclude this column (but keep all the others).

Reply:
Got it working on a local run, and it is running in IRIDA-Next local as well. But I don't where to look to see the metadata information for a run that is saved to confirm. Based on the `iridanext.output.json.gz` I would say it should be good now though.